superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -58,6 +58,11 @@ import { ChartState, ExplorePageState } from 'src/explore/types';
 import { Tooltip } from 'src/components/Tooltip';
 
 import { rgba } from 'emotion-rgba';
+import { kebabCase } from 'lodash';
+import {
+  ExclamationCircleOutlined,
+  InfoCircleOutlined,
+} from '@ant-design/icons';
```

Comment:
We don't import icons directly from Antdesign. You need to use the _Icons_ component that comes with Antdesign icons built-in as well

Reply:
Oops thanks!  I've done that a couple of other times, I can open another PR that fixes the others.

Reply:
If those are not too many, since we have to change them here, it might be a good idea to change them in this PR but I am happy to change the others in a separate PR as long as we fix the ones in this PR

Reply:
No worries I'll do it here!

Reply:
Done – had to add a `font-size: unset;` to uses because the `Icons` component defaults to `font-size: 24px;`.

Reply:
Actually, it turns out the directly-imported AntD icons render different HTML/styles from the `Icons`-rendered ones: there's an extra wrapper element and it resets the font size.  I found 10 or so instances of directly importing AntD icons and swapping them out caused a bunch of layout bugs.  I need to look into this further to try to fix the layout bug you found above, because I think that may be caused by my inconsistently switching to direct AntD import in some places in this PR, but I think switching them out across the codebase will be a more involved process that might be better done in a different PR.